src/SFA.DAS.FAT.Web/Views/Courses/CourseProviders.cshtml

```diff
@@ -16,6 +19,12 @@
         CourseDescription = Model.Course.TitleAndLevel
     };
     ViewData["ShortlistItemCount"] = Model.ShortlistItemCount;
+
+    var clearFiltersRouteData = new Dictionary<string, string>
```

Comment:
Keep on seeing this, and keep on being reminded that we need to do a generic extension for building these various route data dictionaries from the model